archimedes/Makefile

```diff
@@ -1,12 +1,10 @@
 include ../make-base/stubs.mk
-
-app_name=archimedes
 include ../make-base/docker-compose.mk
 
 app_name_capitalized:=$(shell echo ${app_name} | tr [a-z] [A-Z])
 app_service_name:=${app_name}_app
 baseTag:=$(shell basename "$$(pwd)")
 fullTag:=$(IMAGES_PREFIX)$(baseTag)$(IMAGES_POSTFIX)
 
-release-test:: docker-ready
+run-image-test:: docker-ready
```

Comment:
`release-test` now defers to a new target, `run-image-test`, if and only if the image.marker is udpated and requires a test execution to occur.  This is a simplification of the previous weird ass manipulation I was doing for similar effect, see below.